### PR TITLE
Update picodrive.md

### PR DESCRIPTION
**Pull request by CuttleFishJones8652**
- Sync PicoDrive with template
- Update images
- Misc fixes

### CODE REVIEW

File: docs/library/picodrive.md [hunk 1]
```diff
@@ -117,14 +124,14 @@ If you have e.g. `foo.bin`, you should create a text file and save it as `foo.cu
    INDEX 01 00:00:00
 ```
 
-After that, you can load the `foo.cue` file in RetroArch with the PicoDrive core.
+After that, you can load the `foo.cue` file in RetroArch with the Genesis Plus GX core.
```
Comment (hizzlekizzle) [inline docs/library/picodrive.md]: Why was this changed to genplusgx?

File: docs/library/picodrive.md [hunk 2]
```diff
@@ -44,24 +52,13 @@ RetroArch database(s) that are associated with the PicoDrive core:
 - [Sega - PICO](https://github.com/libretro/libretro-database/blob/master/rdb/Sega%20-%20PICO.rdb)
 - [Sega - 32X](https://github.com/libretro/libretro-database/blob/master/rdb/Sega%20-%2032X.rdb)
 
-## BIOS
```
Comment (hizzlekizzle) [inline docs/library/picodrive.md:47]: ~~why are these BIOS checksums removed?~~ nvm, I see that they were just moved.

File: docs/library/picodrive.md (hunk 1 above)
Comment (CuttleFishJones8652) [reply docs/library/picodrive.md]: Ah, I was using the Genesis Plus GX  doc as a template and forgot to replace that. I pushed a commit to fix it.
